### PR TITLE
Remove rowcol labels

### LINKED ISSUE #752: Proposal: Remove row and column labels
### Description

Anecdotally, arbitrary labelling of rows and columns seems to be more confusing than useful.
 I'm talking about when you create an index and frame, and you can optionally specify a column label an row label respectively to be used in queries in place of "rowID" and "columnID". Normally to retrieve a row or column, you'd use `Bitmap(rowID=<id>, frame=<frame>)` or `Bitmap(columnID=<id>, frame=<frame>)`, but if you specified row and column labels, you use those in place of "rowID" and "columnID".

The only value this provides is making queries read in a more relatable way to the data in Pilosa. On the flipside, it increases code complexity (both client and server side), makes it difficult to understand whether a query is accessing a row or column at a glance, and increases cognitive load in that you have to remember which label was used for each frame and index. It also may fragment the language which different groups use to discuss Pilosa and inhibit understanding between different users.

Therefore, in order to simplify Pilosa, and reduce our API surface area, I propose that we deprecate and eliminate the ability to specify arbitrary labels on rows and columns.

The purpose of this ticket is to gather community feedback to determine the level of disruption which this change would cause, and whether we don't understand any important uses of row and column labels.

### Success criteria (What criteria will consider this ticket closeable?)
Row and column labels have been removed from Pilosa, the documentation, and any examples or tools which were using them.

### PR DESCRIPTION
**Pull request by travisturner**
## Overview

This PR removes support for `RowLabel` and `ColumnLabel` and instead uses `row` and `col` in PQL.

Fixes #752 

## Pull request checklist

- [x] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [x] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [x] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [x] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [x] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: frame.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 
 // Default frame settings.
 const (
-	DefaultRowLabel       = "rowID"
+	DefaultRowLabel       = "row"
```
Comment (jaffee) [inline frame.go]: shouldn't this setting not exist anymore?

File: index.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ import (
 
 // Default index settings.
 const (
-	DefaultColumnLabel = "columnID"
+	DefaultColumnLabel = "col"
```
Comment (jaffee) [inline index.go]: again, this goes away, right?

File: internal/private.proto [hunk 3]
```diff
@@ -3,12 +3,10 @@ syntax = "proto3";
 package internal;
 
 message IndexMeta {
-	string ColumnLabel = 1;
```
Comment (jaffee) [inline internal/private.proto]: we'll need to try to synchronize client updates as well when we merge this, right?

File: frame.go (hunk 1 above)
Comment (travisturner) [reply frame.go]: Well, the constant needs to be defined somewhere (unless we want the string "row" scattered throughout the code). Is it that the name of the constant is confusing?  Perhaps changing it to `RowLabel` or something else makes more sense.

Comment (travisturner) [reply frame.go]: I could also move those constants to `executor.go` since that's the only place they're used anymore.

Comment (jaffee) [reply frame.go]: +1 on name change and moving

Comment (travisturner) [reply frame.go]: moved and renamed in e3cd03d902174524de1082cff72c609b9342b8f6

**Comment by jaffee** [comment]
nice finds @alanbernstein

**Comment by travisturner** [comment]
Thanks @alanbernstein 
I removed most of those in ce4fb3c4d347eb66bc0682ca27592e73c2708c87
There was one case where those terms are still valid (in `ast.go`, in the function that determines if the query is meant for the inverse view), so I left that. Also, the instances in `handler_internal_test.go` where ok because they were meant to check for an invalid argument (which `rowLabel` and `columnLabel` are now), but I just changed the string in the test to `badKey` to avoid confusion.

File: internal/private.proto (hunk 3 above)
Comment (jaffee) [reply internal/private.proto]: my fear is unfounded here... this is just internal comms

**Pull request merged**